GNUmakefile

```diff
@@ -451,5 +451,19 @@ gnulib-update:
 	  --source-base=lib/gnu --lgpl=2 --no-vc-files --no-conditional-dependencies \
 	  $(GNU_MODS_AVOID:%=--avoid %) --import $(GNU_MODS)
 
-ancillary-clean: spec-clean srpm-clean mock-clean coverity-clean
+## Coverage/profiling
+
+.PHONY: coverage
+coverage: core
+	-find . -name "*.gcda" -exec rm \{\} \;
+	$(MAKE) $(AM_MAKEFLAGS) check
+	gcovr -r . --html -o coverage.html -e '.*_test.c' -e 'replace'
+
+.PHONY: coverage-clean
+coverage-clean:
+	-rm coverage.html
+	-find . -name "*.gcno" -exec rm \{\} \;
+	-find . -name "*.gcda" -exec rm \{\} \;
```

Comment:
I prefer rm -f in Makefiles, highly unlikely to fail or complain. Also, -o is POSIX so you can say -name ... -o -name ...

Reply:
Fixed.